### DISCUSSION TITLE
请问微信开发者工具可以用unocss吗

### DISCUSSION THREAD

**Discussion by FN20200222** [unocss/unocss#1594] [Q&A]

**Comment by MellowCo** [comment]
可以使用 [@unocss/cli](https://github.com/unocss/unocss/tree/main/packages/cli)  ,监听生成 小程序css, [参考demo](https://github.com/MellowCo/unocss-wechat)

**Comment by SnowingFox** [comment]
直接用这个吧[preset](https://github.com/4chao/preset)，什么配套设施都有，开箱即用
